.cbmc-batch/jobs/aws_cryptosdk_enc_ctx_init/README.md

```diff
@@ -0,0 +1,34 @@
+# Memory safety proof for aws_cryptosdk_enc_ctx_init
+
+This proof harness attains 91% code coverage.  The following comments explain
+why the uncovered lines of code are unreachable code.
+
+Some functions contain unreachable blocks of code:
```

Comment:
Are these intrinsic to the function, or limitations of the harness?

Reply:
These are intrinsic to the function the way it is currently written. I have added more details in the README about why code is unreachable.

Reply:
Sounds good, and the documentation will help with future efforts!